chef/cookbooks/ceph/recipes/conf.rb

```diff
@@ -6,7 +6,14 @@
 
 mon_init = []
 mon_nodes.each do |monitor|
-    mon_init << monitor.name.split(".")[0]
+  mon_name = monitor.name.split(".")[0]
```

Comment:
sure, but you could just pass in the node as a parameter, and then it would work, e.g. 

```ruby
    get_node_name(monitor)
```